pypeit/scripts/coadd_2dspec.py

```diff
@@ -31,17 +13,9 @@ def get_parser(cls, width=None):
         parser = super().get_parser(description='Coadd 2D spectra produced by PypeIt',
                                     width=width)
 
-        parser.add_argument("--file", type=str, default=None, help="File to guide 2d coadds")
-        parser.add_argument('--det', default=None, type=str,
-                            help="1-indexed detector or list of detectors that the user wants to"
-                                 "coadd. If None, all the detectors are coadded. If the spec2d are"
```

Comment:
I am not happy with the --obj method disappearing. You are going to need to convince me it is necessary.

Reply:
@jhennawi the new script `pypeit_setup_coadd2d` creates a `.coadd2d` file for each `obj`. So, you just need to run `pypeit_setup_coadd2d` and then run `pypeit_coadd_2dspec` using the `.coadd2d` file for the object you are interest in. This is why, I think the --obj option is not necessary anymore. This is my interpretation, but I'm sure @kbwestfall can tell you his vision.

Reply:
That's right.  I talked this over with @jhennawi this morning, and I think he's now okay with this (if maybe begrudgingly so...).